client/components/document/DocumentDetails.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import renderHTML from 'react-render-html';
+
+class DocumentDetails extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Again, since you are not doing any special transformation to your props, there would be no need for state. Just let your props go through and use them directly.